07-advanced-git/07-01-git-log/README.md

```diff
@@ -0,0 +1,121 @@
+# Git Log
+
+Now that you know the basics of Git and have started working on the repository with a few commits, you might ask yourself, "How do we visualize these commits and changes we have made in the repository?"  
+
+This is where this chapter, and the subsequent (`Git Diffs`), come in. These commands give you a way to visualize the commits and changes made in your Git repository.  
+
+**TL|DR:** The command `git log` and its corresponding optional flags lets you look at the history of commits and how they link up in your repository.  
```

Comment:
```suggestion
**TLDR:** The command `git log` and its corresponding optional flags lets you look at the history of commits and how they link up in your repository.  
```
![image](https://user-images.githubusercontent.com/34702921/87403414-6a548100-c5ef-11ea-9e5b-98299f76b8a0.png)

Seems like github pages does not render this properly, changing it to just be bolded